packages.el

```diff
@@ -32,7 +32,7 @@
 
 (defun coq/init-company-coq ()
   (use-package company-coq
-    :defer t
+    :mode ("\\.v\\'" . coq-mode)
```

Comment:
It's not problematic but is confusing, given that use-package generally doesn't require an explicit `:defer t`: https://github.com/jwiegley/use-package#notes-about-lazy-loading.